### ISSUE TITLE
An issue about training data

### ISSUE THREAD

**Issue by SiqiHan1012** [zacharycbrown/ssl_baselines_for_biosignal_feature_extraction#2]
Hi, thanks for your wonderful work! It helps me a lot. By the way, could you tell me which dataset you used in this work?

**Comment by zacharycbrown** [comment]
Of course, glad to hear it! Unfortunately, the dataset I was using is a
private dataset owned by someone else, so I cannot release it. I would be
happy to try and answer any questions you may have with regards to adapting
the repo to fit your needs though!

**Comment by SiqiHan1012** [comment]
OK, I understand, thanks again! 
Could you tell me the input shape of Class RPDataset at least? It will help me to preprocess my dataset to the fit the network.
And I am not sure that the _TIME.mat files are necessary or not. If they are necessary, how can I get them and the _LFP.mat files?

**Comment by zacharycbrown** [comment]
Sure!

If I understand your first question correctly, when you ask about "the input shape for the class RPDataset", you are wondering what size the tensors returned by the data loader will be when training on the RP task, is that right? If so, then the tensors returned by the RP data loader (x1, x2, y) should be  (batch_size, num_timesteps, num_channels), (batch_size, num_timesteps, num_channels), and (batch_size) respectively if I remember correctly. Also, if my memory is accurate, the current StagerNet implementation in models.py assumes that num_timesteps=3000 and num_channels=11.

As far as the _TIME.mat files go, please see https://github.com/zacharycbrown/ssl_baselines_for_biosignal_feature_extraction/issues/1, and particularly https://github.com/zacharycbrown/ssl_baselines_for_biosignal_feature_extraction/issues/1#issuecomment-853850812

Does that answer both of your questions?

**Comment by SiqiHan1012** [comment]
Thanks a lot, I can already use my dataset on the model. 
I will come back to you if I have more questions in the future lol.

**Comment by zacharycbrown** [comment]
Sure! Glad to help
